explorer/explorer.go

```diff
@@ -249,10 +308,17 @@ func New(dataSource explorerDataSourceLite, primaryDataSource explorerDataSource
 		}
 	}
 
-	if !exp.liteMode {
+	// Do not fetch charts updates when on liteMode or when blockchain syncing
+	// is running in the background.
+	exp.NewBlockDataMtx.RLock()
+
+	isSyncRunning := exp.DisplaySyncStatusPage || SyncExplorerUpdateStatus()
```

Comment:
If an `atomic.Value` is not an attractive solution (it has drawbacks), how about:

```go
func (exp *explorerUI) DisplaySyncStatusPage() bool {
	exp.SomeMtx.RLock()
	defer exp.SomeMtx.RUnock()
	return exp.displaySyncStatusPage
}
```

This is much less error prone.